framework/engine/DecisionEngine.py

```diff
@@ -6,7 +6,6 @@
 if this environment variable is not defined the ``DE-Config.py`` file from the ``../tests/etc/` directory will be used.
 """
 
-import argparse
```

Comment:
Some of the changes here go beyond addressing the immediate issue, and they require discussion.  For example:

- Do we really want to get rid of program options?  I've actually been thinking of expanding on their use
- The server must be able to be started without requiring the presence of any channels; we will soon be adding functionality so that channels can be added after the server has started

Reply:
I'll revert those bits.

Reply:
Can we have discussion on RB when it appears there please?